processing/src/main/java/org/apache/druid/segment/join/table/IndexedTableJoinMatcher.java

```diff
@@ -81,6 +81,7 @@
   )
   {
     this.table = table;
+    this.currentRow = -1;
```

Comment:
Perhaps create a named constant for this that's also used by the `reset()` method in this class

Reply:
Added a constant